hack/verify-build.sh

```diff
@@ -20,15 +20,6 @@ set -o pipefail
 
 PLATFORMS=(
     linux/amd64
-    linux/386
-    linux/arm
-    linux/arm64
-    linux/ppc64le
-    linux/s390x
-    windows/amd64
-    windows/386
-    freebsd/amd64
-    darwin/amd64
```

Comment:
Maybe we can keep darwin and windows/amd64, but I'm really wondering if we need the others.

Reply:
Let's keep `darwin/amd64`, `linux/amd64` and possibly `linux/arm64`. (don't know if anyone uses kepctl on a Raspberry Pi or a similar device 😅)

Wondering if we should add `darwin/arm64` as well? Many folks are moving to ARM Macs, although I don't have any metrics to prove the same.

Reply:
As an addition, I am feeling like we should expose an env var which let's users the flexibility with specifying what they want to build for. Maybe `$PLATFORM` itself?

Reply:
> Let's keep `darwin/amd64`, `linux/amd64` and possibly `linux/arm64`. (don't know if anyone uses kepctl on a Raspberry Pi or a similar device :sweat_smile:)

Raspis are `armhf` or 32-bit ARM so this is N/A :) Most arm64 devices are servers, we probably can skip it. But we should probably keep Windows.

Reply:
I wouldn't add darwin/arm64 as it wasn't there and no one actually missed it :P

But anyway keeping darwin/amd64, linux/amd64 and windows/amd64 only is already a win. 

About adding the variable, I'm not against it, just wondering if we want this to be a multi arg variable, or if a single one (like BUILD_PLATFORM=$1) and verify if this is not empty, use it otherwise use the pre-populated PLATFORMS.

WDYT?

Reply:
> Raspis are armhf or 32-bit ARM so this is N/A :)

Ah! E_TOO_MANY_ARCHS 😅 

> I wouldn't add darwin/arm64 as it wasn't there and no one actually missed it :P

LOL. Makes sense! If someone wants to, we can add later.

> About adding the variable, I'm not against it, just wondering if we want this to be a multi arg variable, or if a single one (like BUILD_PLATFORM=$1) and verify if this is not empty, use it otherwise use the pre-populated PLATFORMS.

The best option is a multi-arg variable, then a single variable with comma-separated values. I'm fine with either option. Also, pondering more over it, I'm thinking we can lazily add this feature if someone needs it in future.

Reply:
sounds good to me :D 

Let me push with darwin/amd64 and windows/amd64 and let me know if you want me to change anything else here :D

Reply:
All okay from my side. Thank you for the changes!